lib/replica.rb

```diff
@@ -79,179 +85,181 @@ def self.parse_revision(revision)
   # As of 2015-02-24, revisions.php only queries namespaces:
   #   0 ([mainspace])
   #   2 (User:)
-  def self.get_revisions_raw(users, rev_start, rev_end, language=nil)
+  def get_revisions_raw(users, rev_start, rev_end)
+    # TODO: waiting for the backend change
+    # user_list = compile_usernames_string(users)
     user_list = compile_user_ids_string(users)
     oauth_tags = compile_oauth_tags
     oauth_tags = oauth_tags.blank? ? oauth_tags : "&#{oauth_tags}"
     query = user_list + oauth_tags + "&start=#{rev_start}&end=#{rev_end}"
-    api_get('revisions_by_user_id.php', query, language)
+    api_get('revisions_by_user_id.php', query)
   end
 
   # Given a list of users, fetch their global_id and trained status. Completion
   # of training is defined by the users.php endpoint as having made an edit
   # to a specific page on Wikipedia:
   # [[Wikipedia:Training/For students/Training feedback]]
-  def self.get_user_info(users, language=nil)
+  def get_user_info(users)
+    # TODO: usernames, see above
     query = compile_user_ids_string(users)
     if ENV['training_page_id']
       query = "#{query}&training_page_id=#{ENV['training_page_id']}"
     end
-    api_get('users.php', query, language)
+    api_get('users.php', query)
   end
 
   # Given a list of articles, see which ones have not been deleted.
-  def self.get_existing_articles_by_id(articles, language=nil)
-    article_list = compile_article_id_string(articles)
-    existing_articles = api_get('articles.php', article_list, language)
+  # FIXME: inconsistent signature
+  def get_existing_articles_by_id(page_ids)
+    article_list = compile_article_id_string(page_ids)
+    existing_articles = api_get('articles.php', article_list)
+    # FIXME: What does this do?
```

Comment:
The `unless` statement was left over from a refactor: https://github.com/WikiEducationFoundation/WikiEduDashboard/commit/0865d9a301913da0e0eb6c41c64aab1f31ad6cd2